sdk/cosmos/dev.md

```diff
@@ -30,7 +30,7 @@ mvn test -DACCOUNT_HOST="https://REPLACE_ME_WITH_YOURS.documents.azure.com:443/"
 ## Import into Intellij or Eclipse
 
 - Load the main parent project pom file in Intellij/Eclipse (That should automatically load examples).
-- For running the samples you need a proper Azure Cosmos DB Endpoint. The endpoints are picked up from [TestConfigurations.java](examples/src/test/java/com/microsoft/azure/cosmosdb/rx/examples/TestConfigurations.java). There is a similar endpoint config file for the sdk tests [here](sdk/src/test/java/com/microsoft/azure/cosmosdb/rx/TestConfigurations.java).
+- For running the samples you need a proper Azure Cosmos DB Endpoint. The endpoints are picked up from [TestConfigurations.java](https://github.com/Azure/azure-sdk-for-java/blob/master/sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/TestConfigurations.java).
```

Comment:
Does the tool support relative markdown links? It'll be easier to read imho.

Reply:
We are going to stop the support for relative links to files - https://github.com/Azure/azure-sdk/blob/master/docs/policies/README-TEMPLATE.md We have updated the guidelins here